index.html

```diff
@@ -293,8 +293,38 @@ <h4><a href="#">LET'S CONNECT </a></h4>
             </ul>
         </div>
     </section>
-</div>
-
-   
+    <!-- This is the footer section of the webpage-->
+    <footer>
+        <section id="contact-form">
+            <h5>Contact me</h5>
+            <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+            </p>
+            <form class="input" action="https://formspree.io/f/xlezvgbv" method="post">
+               <input type="text" class="input-item" name="Name" placeholder="full name" maxlength="30">
+               <input type="email" class="input-item" name="Email" placeholder="email">
+               <textarea name="message" cols="30" rows="10" placeholder="write your message here" maxlength="500"></textarea>
```

Comment:
- [ ] [REQUIRED] Kindly ensure all input fields have the required attribute to activate the validation on submit. This is a core requirement for this project.